descriptors/ModuleDescriptor-template.json

```diff
@@ -594,6 +594,34 @@
           ],
           "unit": "minute",
           "delay": "5"
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/circulation/request-scheduled-notices-processing",
+          "modulePermissions": [
+            "scheduled-notice-storage.scheduled-notices.collection.get",
+            "scheduled-notice-storage.scheduled-notices.item.delete",
+            "scheduled-notice-storage.scheduled-notices.item.put",
+            "inventory-storage.items.item.get",
+            "inventory-storage.locations.item.get",
+            "inventory-storage.location-units.institutions.item.get",
+            "inventory-storage.location-units.campuses.item.get",
+            "inventory-storage.location-units.libraries.item.get",
+            "inventory-storage.material-types.item.get",
+            "inventory-storage.holdings.item.get",
+            "inventory-storage.loan-types.item.get",
+            "inventory-storage.service-points.item.get",
+            "inventory-storage.instances.item.get",
+            "circulation-storage.loans.collection.get",
+            "circulation-storage.requests.item.get",
+            "patron-notice.post",
+            "users.item.get",
+            "usergroups.collection.get"
+          ],
+          "unit": "minute",
+          "delay": "2"
```

Comment:
Why do request notices need to be sent more frequently?

Reply:
It has been decided because currently notice processing took less than a minute. Running the job more frequently makes sense to proccess more notices. I suppose that the interval for due date notices also will be changed later.

Reply:
Under what circumstances does it take less than a minute?

Could it take longer? What happens if it does?

Reply:
In experimental way we found that it takes less than a minute when 100 notices are processed (for now it is hardcoded contant). The performance test is going to be written for this functionality. Depending on the result of performance tests the interval can be changed in the future.